### PR TITLE
Ads 598 - Moving ads styling from next apps to next-ads-component

### PR DESCRIPTION
**Pull request by leggsimon**
@robin-marr-ft @andrewgeorgiou1981 @VladDubrovskis

### CODE REVIEW

File: client/main.scss [hunk 1]
```diff
@@ -52,8 +52,6 @@ $o-comments-is-silent: false;
 	.n-myft-tray__cta,
 	.n-myft-tray,
 	.next-header,
-	.article__leaderboard-advert,
```
Comment (andygnewman) [inline client/main.scss]: presume you're putting the @media print {display: none} for the ad units into next-ads-component?

Comment (leggsimon) [inline client/main.scss]: Yes, it's done here https://github.com/Financial-Times/next-ads-component/blob/ADS-598/src/scss/_print.scss

Comment (andygnewman) [inline client/main.scss]: Great, thanks

File: views/article.html [hunk 2]
```diff
@@ -9,7 +9,6 @@
 
 <article id="site-content" role="main" class="article{{#if designGenre}} article--brand {{#if designGenre.headshot}}article--brand--headshot{{/if}}{{/if}}" data-content-id="{{id}}" data-content-sources="{{#if articleV1}}v1{{/if}} {{#if articleV2}}v2{{/if}}" data-trackable="article" data-trackable-terminate>
 	<div class="article__header" data-trackable="header">
-		<div class="article__leaderboard-advert"></div>
```
Comment (andygnewman) [inline views/article.html]: Is there no placeholder in the new version for a header leaderboard?  eg. .header-ad-placeholder__bottom

Comment (leggsimon) [inline views/article.html]: So the leaderboard ads in article and stream pages are now a property of `n-header-footer` where it places an ad slot below the nav bar

Comment (andygnewman) [inline views/article.html]: OK, sensible approach.

**Comment by andygnewman** [comment]
:+1:

**Comment by andrewgeorgiou1981** [comment]
:thumbsup:

**Pull request merged**